edm4eic.yaml

```
@@ -544,3 +544,15 @@ datatypes:
      - edm4eic::RawTrackerHit rawHit        // reference to the digitized hit
     OneToManyRelations:
      - edm4hep::SimTrackerHit simHits       // reference to the simulated hits
+
+
+  edm4eic::MCParticleRecoHitAssociation:
+    Description: "Association between a TrackerHit and MCParticles. One tracker hit can be linked to multiple simulation hits hence multiple particles"
+    Author : "S. Li"
+    VectorMembers:
+      - uint32_t            simIDs          // Index of corresponding MCParticles (position in MCParticles array)
+      - float               weights         // weight of this association
```

This is getting strange. The original motivation for keeping simID was to keep poor users from having to access branches with strange names. In this case, using vector members will create `CollectionName.simIDs_begin`, `CollectionName.simIDs_end` and `_CollectionName_simIDs`, so a user would need to do two dereferences instead of one with just `mcParticles_begin` and `mcParticles_end`.

```suggestion
    VectorMembers:
      - float               weights         // weight of this association
```